build/helper/codegen_helper.py

```diff
@@ -148,10 +148,11 @@ def get_enum_type_check_snippet(parameter, indent):
 
 def _get_buffer_parameter_for_size_parameter(parameter, parameters):
     '''If parameter represents the size of another parameter in the C API, returns that other parameter. Otherwise None.'''
+    buffer_params = []
```

Comment:
I think 151 through the end of the function can be written as a one-line list comprehension... if you're feeling pythonic:
https://www.python.org/dev/peps/pep-0202/